Sources/ArcGISToolkit/Components/FeatureFormView/Subviews/Inputs/TextInput.swift

```diff
@@ -82,24 +78,33 @@ private extension TextInput {
     /// The body of the text input when the element is editable.
     var textWriter: some View {
         HStack(alignment: .bottom) {
-            TextField(
-                element.label,
-                text: $text,
-                prompt: Text(element.hint).foregroundColor(.secondary),
-                axis: element.isMultiline ? .vertical : .horizontal
-            )
-            .disabled(element.isMultiline)
-            .sheet(isPresented: $fullScreenTextInputIsPresented) {
-                FullScreenTextInput(text: $text, element: element)
+            Group {
+                if element.isMultiline {
+                    Text(text)
+                        .accessibilityIdentifier("\(element.label) Text Input Preview")
+                        .fixedSize(horizontal: false, vertical: true)
+                        .lineLimit(10)
+                        .truncationMode(.tail)
```

Comment:
This is a considerable change where we no longer show the entire text by default. Limiting the size makes a significant performance improvement when the form is drawn.

Reply:
I think it will also improve the user experience, as there's no point in showing 28k worth of text; it just makes viewing the rest of the form more difficult.